kintree/gui/views/settings.py

```diff
@@ -732,13 +745,24 @@ class InvenTreeSettingsView(SettingsView):
     ]
 
     def save(self, file=None, dialog=True):
+        address = SETTINGS[self.title]['Server Address'][1].value
+        proxy = SETTINGS[self.title]['Proxy'][1].value
+        enable_proxy = SETTINGS[self.title]['Enable Proxy Support'][1].value
+        if not enable_proxy:
+            proxies = {}
```

Comment:
Could you replace this line with:
``` python
proxies = None
```

else it saves it as `PROXIES: {}` in config file which I don't find very clean